orchagent/orch.cpp

```diff
@@ -301,7 +301,7 @@ bool Orch::bake()
 }
 
 /*
-- Validates reference has proper format which is [table_name:object_name]
+- Validates reference has proper format which is not ABNF [table_name:object_name]
```

Comment:
Emphasizing `not ABNF` looks strange. I think `Validates reference has proper format which is object_name` should be good.

Reply:
Done